stackrox/acs-fleet-manager
# added DNS e2e tests

## Description
Add tests for creating routes when route resource is available and also for creating and deleting CNAME record in AWS Route53 when Route53 credentials are available.

## Checklist (Definition of Done)


## Test manual

- Follow instructions in `howto-run-e2e-tests.md`
- Run against a cluster without openshift routes and make sure DNS and route test is skipped
- Run against a cluster having openshift routes crd using Route53 credentials


```
# To run tests locally run:
make db/teardown db/setup db/migrate
make ocm/setup OCM_OFFLINE_TOKEN=<ocm-offline-token> OCM_ENV=development
make verify lint binary test test/integration
```
